src/main/java/io/github/vcuswimlab/stackintheflow/model/L2H/L2HPredictor.java

```diff
@@ -164,4 +197,13 @@ public void computeL2HPredictions(String fileToPredict, String outputFileName, S
 		}
 		return result;
 	}
+
+	public static void main(String[] args) throws Exception {
```

Comment:
So I'm still confused why there needs to be a public static void main method anywhere in this project. Are you calling this while the plugin is running in the ide?